workflows/config/process_slc_pair/luigi.cfg.template

```diff
@@ -10,8 +10,9 @@ paths = {"input":"/input",
             "toolchainXML": "SLCCoh_Scot_CommandLine.xml"
             }
 sourceSRS = 4326
+noCopyState = True
```

Comment:
Don't copy state to output folder by default - currently don't have a need for them unless there's an error in which case they'll be in the working folder.